src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -111,33 +113,53 @@ class CodyAgentService(project: Project) : Disposable {
   }
 
   companion object {
+    private val logger = Logger.getInstance(CodyAgent::class.java)
+
+    val agentError: AtomicReference<String?> = AtomicReference(null)
```

Comment:
nit: `s/agentError/mostRecentlyReportedError/`?

Reply:
All of those errors are `recently reported` as they are used in the status bar.
So really I do not think adding it only for this element (or at all) adds any information here.

Reply:
no strong feelings. my suggestion seems to better reflect the variable sense